### PR TITLE
Express delete - Frank Bishop

### PR DESCRIPTION
**Pull request by FrankBishop**
Screenshot (335)
Screenshot (336)

### CODE REVIEW

File: express-delete/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    score: 92
+  }
+};
+
+app.get('/api/grades', function (req, res) {
+  let key;
+  array = [];
+  for (key in grades) {
+    array.push(grades[key]);
+  }
```
Comment (thebearingedge) [inline express-delete/index.js:30]: Your array should only be declared in here where it's needed.

File: express-delete/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  res.json(array);
+  grades = array;
+  console.log(grades);
+});
+
+app.delete('/api/grades:id', function (req, res) {
+  res.send(req.params);
```
Comment (thebearingedge) [inline express-delete/index.js:37]: You don't need to send the `req.params` back to the client.

File: express-delete/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  grades.splice(req.params, 1);
```
Comment (thebearingedge) [inline express-delete/index.js:38]: Try using the `delete` operator on your `grades` object like you did in `node-notes-cli`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by FrankBishop** [comment]
Somehow other files got added to this when I pushed it and I can't find the correct one that was supposed to get pushed on GitHub.  We can look into it tomorrow.

Thanks

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**